_includes/head.html

```diff
@@ -8,7 +8,7 @@
 
   {% assign custom_url = site.url | append: site.baseurl %}
   {% assign full_base_url = custom_url | default: site.github.url %}
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: full_base_url }}">
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | prepend: full_base_url }}">
```

Comment:
> Why do you say that? Why does it have to be?

That's my understanding.. because when jekyll builds the site, the stylesheet will be in `_site/assets/css` instead of `_site/css` 

--
/cc @parkr